docs/source/install.rst

```diff
@@ -174,4 +111,4 @@ Optional Dependencies
   docs locally or to use the notebooks.
 * `joblib <http://pythonhosted.org/joblib/>`__ >= 0.9 can be used to accelerate distributed
   estimation for certain models.
-* `jupyter <http://jupyter.org/>`__ is needed to run the notebooks.
\ No newline at end of file
+* `jupyter <http://jupyter.org/>`__ is needed to run the notebooks.
```

Comment:
Wasn't there a thing recently about all-https?  Not sure if that's relevant here.

Reply:
I think that only applies to images/resources auto-loaded by the webpage. Not links for the user.